manifests/init.pp

```diff
@@ -203,31 +203,45 @@
         ]
         $default_ssh_config_hash_known_hosts     = 'yes'
         $default_sshd_config_xauth_location      = undef
-      } else {
-        $default_sshd_config_hostkey             = [ '/etc/ssh/ssh_host_rsa_key' ]
-        $default_ssh_config_hash_known_hosts     = 'no'
-        $default_sshd_config_xauth_location      = '/usr/bin/xauth'
       }
       $default_packages                        = ['openssh-server',
                                                   'openssh-client']
       $default_service_name                    = 'ssh'
-      $default_ssh_config_forward_x11_trusted  = 'yes'
-      $default_ssh_package_source              = undef
-      $default_ssh_package_adminfile           = undef
-      $default_ssh_sendenv                     = true
-      $default_sshd_config_subsystem_sftp      = '/usr/lib/openssh/sftp-server'
-      $default_sshd_config_mode                = '0600'
-      $default_sshd_config_use_dns             = 'yes'
-      $default_sshd_use_pam                    = 'yes'
-      $default_sshd_gssapikeyexchange          = undef
-      $default_sshd_pamauthenticationviakbdint = undef
-      $default_sshd_gssapicleanupcredentials   = 'yes'
-      $default_sshd_acceptenv                  = true
-      $default_service_hasstatus               = true
-      $default_sshd_config_serverkeybits       = '1024'
-      $default_sshd_addressfamily              = 'any'
-      $default_sshd_config_tcp_keepalive       = 'yes'
-      $default_sshd_config_permittunnel        = 'no'
+
+      case $::facts['os']['release']['major'] {
```

Comment:
this code is against puppet3 .. use `$::operatingsystemrelease` or similar fact.

Reply:
On my box this report not the valume i'm looking for : 
```
$ facter -p operatingsystemrelease
9.8
```
Instead I want to grab the major version. 
```
$ facter -p os
{
  architecture => "amd64",
  distro => {
    codename => "stretch",
    description => "Debian GNU/Linux 9.8 (stretch)",
    id => "Debian",
    release => {
      full => "9.8",
      major => "9",
      minor => "8"
    }
  },
  family => "Debian",
  hardware => "x86_64",
  name => "Debian",
  release => {
    full => "9.8",
    major => "9",
    minor => "8"
  },
  selinux => {
    enabled => false
  }
}

```
I'm running facter 5.5.13
according to https://puppet.com/docs/facter/3.11/core_facts.html#operatingsystemrelease  `$::operatingsystemrelease`  is deprectad